sklearn/gaussian_process/gaussian_process.py

```diff
@@ -618,14 +618,7 @@ def reduced_likelihood_function(self, theta=None):
 
         # Get generalized least squares solution
         Ft = linalg.solve_triangular(C, F, lower=True)
-        try:
-            Q, G = linalg.qr(Ft, econ=True)
```

Comment:
econ=True removed in scipy=0.9 which matches our minimum scipy version. From 
https://github.com/scipy/scipy/blob/master/doc/release/0.9.0-notes.rst#removed-features.